cri/v1alpha1/cri.go

```diff
@@ -689,7 +689,27 @@ func (c *CriManager) ListContainerStats(ctx context.Context, r *runtime.ListCont
 
 // UpdateContainerResources updates ContainerConfig of the container.
 func (c *CriManager) UpdateContainerResources(ctx context.Context, r *runtime.UpdateContainerResourcesRequest) (*runtime.UpdateContainerResourcesResponse, error) {
-	return nil, fmt.Errorf("UpdateContainerResources Not Implemented Yet")
+	containerID := r.GetContainerId()
+	container, err := c.ContainerMgr.Get(ctx, containerID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get container of %q: %v", containerID, err)
+	}
+
+	// Do not update the container when there is a removal in progress.
+	if container.State.Status == apitypes.StatusRemoving {
+		return nil, fmt.Errorf("container %q is in removing state", containerID)
```

Comment:
This error is not so readable. How about `cannot update container resource when it is in removing state`?